server/routes/admin.js

```diff
@@ -4,17 +4,22 @@ import middlewares from '../middlewares';
 
 
 const {
-  verifyToken, isActive, isJustAUser, validateRoleInput, validateParamsInput
+  verifyToken, isActive, isJustAUser, validateRoleInput, validateParamsInput, validateNewUser,
+  validateUpdateUser, validateGetUser
 } = middlewares;
 
 const {
-  assignRole, setActiveStatus, deleteAUser, deleteAnArticle, deleteAComment, getASingleUser
+  assignRole, setActiveStatus, deleteAUser, deleteAnArticle, deleteAComment, getASingleUser,
+  createUser, updateUser, getUser
 } = AdminController;
 const adminRoute = express();
 
 adminRoute.put('/assignRole', verifyToken, isActive, isJustAUser, validateRoleInput, assignRole);
 adminRoute.put('/setActiveStatus/:username', verifyToken, isActive, isJustAUser, validateParamsInput, setActiveStatus);
 adminRoute.delete('/deleteUser/:username', verifyToken, isActive, isJustAUser, validateParamsInput, deleteAUser);
+adminRoute.put('/user/', verifyToken, isActive, isJustAUser, validateNewUser, createUser);
```

Comment:
Nice Job @Lundii, However, I see that you have used both a `PUT` and a `PATCH` request. Is there a reason why you are using both? Also Why are you using a `PUT` request while creating a user?

Reply:
Thank you for that observation.I was thinking PUT request was the best for the route but after doing further study, I realised it should be a POST request. I have changed it